### ISSUE TITLE
Add Blockchain.info Onion link blockchainbdgpzk.onion in "transaction details" screen

### ISSUE THREAD

**Issue by wodry** [bitcoin-solutions/multibit-hd#774]
0.1.4
On the transaction details screen there is a button "View on blockchain.info". But if one has enabled Tor in the browser, when one clicks the button and the browser opens
https://blockchain.info/tx-index/123456789

there comes only the message from Blockchain.info:
"Please use our onion link https://blockchainbdgpzk.onion/"
This is very uncomfortable. I do not know why blockchain.info, if they detect a Tor user, they do not redirect him to the onion page or why they do not output a html link with the URL path included.

So as this is not the case, it would be nice if on the transaction details screen there would be a second button "View on blockchainbdgpzk.onion (Tor users only)" or something like that.

**Comment by jim618** [comment]
Probably the best way to do this is to  add another selection in the list of available block explorers In Manage wallet | Payments.

**Comment by gary-rowe** [comment]
Agreed. Ultimately we want a complete end to end Tor experience to be available so having a Tor-base block explorer would assist with this.
